src/views/email-2022.js

```diff
@@ -14,10 +13,10 @@ const links = (data) => ({
 })
 
 const images = {
-  header: `${AppConstants.SERVER_URL}/img/email_images/person-at-desk.png`,
-  footer: `${AppConstants.SERVER_URL}/img/email_images/mozilla-logo-bw-rgb.png`,
-  logoDark: `${AppConstants.SERVER_URL}/img/email_images/monitor-logo-transparent-dark-mode.png`,
-  logoLight: `${AppConstants.SERVER_URL}/img/email_images/monitor-logo-bg-f9f9fa.png`
+  header: 'images/person-at-desk.webp',
+  footer: 'images/mozilla-logo-bw.webp',
+  logoDark: 'images/monitor-logo-transparent-dark-mode.webp',
+  logoLight: 'images/monitor-logo-bg-light.webp'
```

Comment:
Hmm... I think we need to use an absolute path because the images are loaded in email and not from our website/server?